### ISSUE TITLE
DockerNetworkService swapped Network ID and Name

### ISSUE THREAD

**Issue by eugendun** [mariotoffia/FluentDocker#85]
Hallo,
 
I wrote a unit test which asserts the presence of a container and found out that the Network ID and the Name of the docker network are swapped on DockerNetworkService.

### DockerNetworkService
![image](https://user-images.githubusercontent.com/2209793/57874437-e3e8b780-7810-11e9-885c-93c5ccb495a6.png)

### docker network ls
![image](https://user-images.githubusercontent.com/2209793/57874619-4772e500-7811-11e9-9d26-d397ab7ad5d0.png)

### mongodb-compose.yml
```yml
version: '3.3'

services:
  mongodb:
    image: mongo:latest
    volumes:
      - mongodb-data:/data
      - /D/data:/backup_data
    command: mongod --smallfiles --bind_ip=0.0.0.0 --logpath=/dev/null
    expose:
      - 27017
    networks:
      - mongodb-network

volumes: 
  mongodb-data:

networks:
  mongodb-network:
```

### Code where is try to resolve the network name of mongodb service
```c#
var file = Path.Combine(Directory.GetCurrentDirectory(), "Compose/mongodb-compose.yml");

services = Fd
  .UseContainer()
  .UseCompose()
  .FromFile(file)
  .KeepOnDispose()
  .KeepVolumes()
  .Build().Start();

var mongodbContainer = services.Services.Where(s => s is IContainerService).Single() as IContainerService;
var networkService = mongodbContainer.GetNetworks().Single();
```

**Comment by mariotoffia** [comment]
Thanks for the great bug report! It's easy to fix when I get good ones like this! :) - Fixed at pushed to nuget - version 2.6.7.

Cheers,
 Mario

**Comment by eugendun** [comment]
Thank you! The project is awesome!
